lib/spack/docs/getting_started.rst

```diff
@@ -13,10 +27,17 @@ Getting spack is easy.  You can clone it from the `github repository
 
    $ git clone https://github.com/llnl/spack.git
 
-This will create a directory called ``spack``.  We'll assume that the
-full path to this directory is in the ``SPACK_ROOT`` environment
-variable.  Add ``$SPACK_ROOT/bin`` to your path and you're ready to
-go:
+This will create a directory called ``spack``.  If you are using Spack
+for a specific purpose, you might have received different instructions
+on how to download Spack; if so, please follow those instructions.
+
+~~~~~~~~~~~~~~~~~~
+Add Spack to Shell
+~~~~~~~~~~~~~~~~~~
```

Comment:
@citibeth Can you replace `~` section underlines/overlines with `^`? That's the convention that Sphinx suggests in its [section documentation](http://www.sphinx-doc.org/en/stable/rest.html#sections). It doesn't really matter what we choose, but if we aren't consistent, you'll see that warning:
```
SEVERE: Title level inconsistent
```
If there are multiple kinds of underlines/overlines used, Sphinx tries to rank all of them separately. This causes the hierarchy to break. At worst, it can cause the section title to not show up at all in the documentation, which was happening before #1576.